website/UNIVERSE.md

```diff
@@ -2,93 +2,103 @@
+✅ Libraries and packages should be **open-source** (with a user-friendly
+license) and at least somewhat **documented** (e.g. a simple `README` with usage
+instructions).
 
-✅ We're happy to include work in progress and prereleases, but we'd like to keep the emphasis on projects that should be useful to the community **right away**.
+✅ We're happy to include work in progress and prereleases, but we'd like to
+keep the emphasis on projects that should be useful to the community **right
+away**.
 
 ✅ Demos and visualizers should be available via a **public URL**.
 
 ### Educational Materials
 
-✅ Books should be **available for purchase or download** (not just pre-order). Ebooks and self-published books are fine, too, if they include enough substantial content.
+✅ Books should be **available for purchase or download** (not just pre-order).
+Ebooks and self-published books are fine, too, if they include enough
+substantial content.
 
-✅ The `"url"` of book entries should either point to the publisher's website or a reseller of your choice (ideally one that ships worldwide or as close as possible).
+✅ The `"url"` of book entries should either point to the publisher's website or
+a reseller of your choice (ideally one that ships worldwide or as close as
+possible).
 
-✅ If an online course is only available behind a paywall, it should at least have a **free excerpt** or chapter available, so users know what to expect.
+✅ If an online course is only available behind a paywall, it should at least
+have a **free excerpt** or chapter available, so users know what to expect.
 
 ## JSON format
 
-To add a project, fork this repository, edit the [`universe.json`](meta/universe.json)
-and add an object of the following format to the list of `"resources"`. Before
-you submit your pull request, make sure to use a linter to verify that your
-markup is correct.
+To add a project, fork this repository, edit the
+[`universe.json`](meta/universe.json) and add an object of the following format
+to the list of `"resources"`. Before you submit your pull request, make sure to
+use a linter to verify that your markup is correct.
 
 ```json
 {
-    "id": "unique-project-id",
-    "title": "Project title",
-    "slogan": "A short summary",
-    "description": "A longer description – *Markdown allowed!*",
-    "github": "user/repo",
-    "pip": "package-name",
-    "code_example": [
-        "import spacy",
-        "import package_name",
-        "",
-        "nlp = spacy.load('en')",
-        "nlp.add_pipe(package_name)"
-    ],
-    "code_language": "python",
-    "url": "https://example.com",
-    "thumb": "https://example.com/thumb.jpg",
-    "image": "https://example.com/image.jpg",
-    "author": "Your Name",
-    "author_links": {
-        "twitter": "username",
-        "github": "username",
-        "website": "https://example.com"
-    },
-    "category": ["pipeline", "standalone"],
-    "tags": ["some-tag", "etc"]
```

Comment:
Oh that's interesting. Prettier for `.mdx` files is setup to indent with two spaces, so this also applies to code within the `.mdx` files. Easiest fix would be to change indentation for `.mdx` to 4 spaces as well, since it's the only file type were we currently make an exception.